src/index.tsx

```diff
@@ -23,47 +23,59 @@ export function faceAuth(capturedImage: string, vcImage: string): Promise<boolea
 export async function init(url: string, overrideCache: boolean): Promise<boolean> {
   try {
     console.log('inside inji face sdk init function, url is - ' + url)
-    const fileDir = `${RNFS.CachesDirectoryPath}/model.tflite`;
-    var fileExists = await RNFS.exists(fileDir);
+    const modelDirPath = `${RNFS.CachesDirectoryPath}/model.tflite`;
+    const checksumCachePath = `${RNFS.CachesDirectoryPath}/model_sha256sum.txt`;
```

Comment:
Can we have the name of the checksum file as `model.sha256`?

Is `model.tflite` a directory or file? If it is a file then the variable should be named as `modelFile` instead of `modelDirPath`?